app/Http/Requests/Tags/DeleteTagRequest.php

```
@@ -0,0 +1,42 @@
+		];
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function authorize(): bool
+	{
+		return Gate::check(TagPolicy::CAN_EDIT, [Tag::class]);
+	}
+
+	protected function processValidatedValues(array $values, array $files): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$files'. (Unused Code Rules)

(UnusedFormalParameter)